### PR TITLE
Support/cleanup for sandbox data-center

### PR DESCRIPTION
**Pull request by j5kay**
- Removed “svcCards” setting
- Centralized API server handling
- Sync common viewer settings, based on adapter settings
-  Fix path to viewer loader (was still pointing glympse.com servers)
- Deleted old beta builds

Requires use of Glympse/web-viewer-gmaps#96 for proper functionality!

PTAL @Glympse/web

### CODE REVIEW

File: app/src/adapter/models/Account.js [hunk 1]
```diff
@@ -387,8 +389,8 @@ define(function(require, exports, module)
 	// Account defines
 
 	// Environment
-	Account.EnvProduction = 'prod';
-	Account.EnvSandbox = 'sbox';
+	Account[cEnvProduction] = 'prod';
+	Account[cEnvSandbox] = 'sbox';
```
Comment (adubovsky) [inline app/src/adapter/models/Account.js]: Why did you change this to use private constants for keys? As I understand these `Account[<keys>]` are publically available, so if we would need to access them from another file, we'd use `Account.<key>` anyway.

But looks we we do not need them outside of Account.js, so can simply remove, and leave just `cEnvProduction='prop'/'sbox'` as private vars.

Comment (j5kay) [inline app/src/adapter/models/Account.js]: It's just a small hack for minification. The Account.Env* "consts" themselves are still public (and used) elsewhere in the adapter

Comment (adubovsky) [inline app/src/adapter/models/Account.js]: Ah, okay.

**Comment by adubovsky** [comment]
LGTM
